apps/researcher/src/lib/community/actions.ts

```diff
@@ -99,6 +50,9 @@ export function sort(communities: Community[], sortBy: SortBy) {
     } else if (sortBy === SortBy.NameDesc) {
       return b.name.localeCompare(a.name);
     } else if (sortBy === SortBy.CreatedAtDesc) {
+      if (typeof a.createdAt !== 'number' || typeof b.createdAt !== 'number') {
+        throw new Error('createdAt must be of type number');
+      }
```

Comment:
Is there a way to do the validation outside of the sort ('separation of concerns')?

Reply:
This is not needed anymore as this is part of the Clerk converter. I will remove this check.